torch/_C/__init__.pyi.in

```diff
@@ -189,6 +189,8 @@ def _jit_pass_convert_frozen_ops_to_mkldnn(graph: Graph): ...
 def _jit_pass_transpose_frozen_linear(graph:Graph): ...
 def _jit_pass_remove_dropout(module: 'torch.jit.ScriptModule'): ...
 
+def _compile_graph_to_code_table(name: str, graph: Graph, compilation_options: CompilationOptions) -> IValue: ...
```

Comment:
remove compilation_options? as in script_init you didnt define it

Reply:
good catch!

Reply:
Yep good catch!